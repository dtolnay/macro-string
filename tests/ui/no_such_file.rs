use macro_string_eval::eval;

const _: &str = eval!(include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/nonexist.ent")));

fn main() {}
