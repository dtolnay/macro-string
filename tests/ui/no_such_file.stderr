error: No such file or directory (os error 2) $DIR/target/tests/trybuild/macro-string/nonexist.ent
 --> tests/ui/no_such_file.rs:3:23
  |
3 | const _: &str = eval!(include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/nonexist.ent")));
  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
